app/views/users/show.html.erb

```diff
@@ -2,4 +2,11 @@
 <%= @user.street_address %>
 <%= @user.city %>
 <%= @user.state %>
-<%= @user.zip %>
+<%= @user.zip %><br>
+
+<% @user.reviews.each do |review| %>
+  <br>
+  Review: <%= review.title %>
```

Comment:
Calling reviews from user instead of a @reviews variable! I think we could use the block variable `review` to call on shelter info if we need to down the lone.